composer.json

```diff
@@ -4,15 +4,13 @@
     "license": "MIT",
     "description": "A pack for functional and end-to-end testing within a Symfony app",
     "require": {
-        "phpunit/phpunit": "^9.5",
```

Comment:
was only restricted to this version because of the bridge: https://github.com/symfony/test-pack/pull/19

Reply:
Is it worth setting `^11.1` here, given that is the first PHPUnit version that is fully compatible with the features provided by PHPUnitBridge's deprecation system? Or would that be redundant, as Composer will automatically pick the latest version anyways?

Reply:
it should then be `>=11.1` IMHO

Reply:
Based on https://github.com/symfony/recipes/pull/1401#issuecomment-2799871703, we may need to set this to some `>=12.x` version then. That could probably mean we have to delay this to Symfony 7.4?

Reply:
How foes the "pack" behave with >= dows it still require it as ^11.1 else a conflict would be better.

Reply:
shouldn't we move this constraint to the conflict section?
conflict with phpunit < 11.1

Reply:
added